install.ps1

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env pwsh
+$python = "python3"
+if ($null -ne (Get-Command python -ErrorAction SilentlyContinue)) {
+    $pythonIsPython3 = (python --version) -match "Python 3.*"
+    if ($pythonIsPython3) {
+        $python = "python"
+    }
+}
+& $python -m pip install -r requirements.txt --no-index --find-links=wheelhouse -v
```

Comment:
Need to update with url to platform package

Reply:
It actually seemed the most reasonable to ask people to download a release using the GitHub CLI, and the command we give will download them into the wheelhouse folder. This script is hence still valid, but the binaries are no longer checked in.  See the installation.md file.